controllers/common.go

```diff
@@ -149,3 +152,47 @@ func checkByteArrayIsEmpty(val []byte) bool {
 
 	return len(val) != 0
 }
+
+func checkForOneDefaultSnapClass(vsClassList *snapv1.VolumeSnapshotClassList, log logr.Logger) (bool, error) {
```

Comment:
non blocking NIT. We are not using log here.. I would rather have this removed from param. Also error is kind of redundant. You already know if you have more than one default snap class by receiving false return value.

There isn't any complex data in the error created here.

Maybe create error in the validator.go instead.

Reply:
```suggestion
func hasOneDefaultSnapClass(vsClassList *snapv1.VolumeSnapshotClassList) bool {
```